core/tests/protractor_utils/forms.js

```diff
@@ -247,11 +248,17 @@ var RichTextEditor = async function(elem) {
       for (var i = 1; i < arguments.length; i++) {
         args.push(arguments[i]);
       }
-      await richTextComponents.getComponent(componentName)
-        .customizeComponent.apply(null, args);
-      await modal.element(
-        by.css('.protractor-test-close-rich-text-component-editor')).click();
-
+      await ((await richTextComponents.getComponent(componentName))
```

Comment:
Could you double-check whether we need the inner `await`? I think `getComponent` is just a map lookup

Reply:
yeah its not needed removed it.